CodeSniffer/Standards/Squiz/Sniffs/PHP/LowercasePHPFunctionsSniff.php

```diff
@@ -119,4 +118,22 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
     }//end process()
 
 
+    /**
+     * Determine whether the functionName in question is a built-in
+     * (or extension-provided) function.
+     *
+     * @param string $functionName Name of the function to check
+     *
+     * @return bool
+     */
+    protected function isBuiltInFunction($functionName)
```

Comment:
Performance-wise this is very bad, because `isBuiltInFunction` is called for every checked function.

Since function list won't change anyway I'm proposing to:

1. create list of functions in sniff constructor and use it here
2. in created function list set function name as key in array and `true` as value
3. check using `isset($this->definedFunctions) === true`

Checking by key is faster, then doing `in_array`.